src/tracker-component.js

```diff
@@ -214,59 +95,26 @@ const getJetlorePayload = (type : string, options : Object) : Object => {
 
 let trackEventQueue = [];
 
-const track = <T>(config : Config, trackingType : TrackingType, trackingData : T) => {
+export const clearTrackQueue = (config : Config) => {
+    // $FlowFixMe
+    return trackEventQueue.filter(([ trackingType, trackingData ]) => { // eslint-disable-line array-callback-return
+        track(config, trackingType, trackingData);
```

Comment:
question, does `track` also fire off JL events?

Reply:
It uses a different 'tracker' method. See "trackEventByType" & "getJetlore".

Reply:
https://github.com/paypal/paypal-muse-components/issues/53